### DISCUSSION TITLE
SSH jump to access through another server

### DISCUSSION THREAD

**Discussion by geonexus** [isontheline/pro.webssh.net#979] [Ideas]
Sometimes it is needed to access an isolated server through another jump server.
Could be great to implement this functionallity, at the end it is an option in ssh/sftp command

ssh -J user@ip_server_A user_2@ip_server_B
sftp -J user@ip_server_A test.txt user2@ip_server_B

BR

**Comment by isontheline** [comment]
Hello @geonexus 👋

Do you have found the solution you was looking for?

Have a nice day ☀️

**Reply by Joegeep** [reply]
I wanna know how to hook up to my server your server use the server on iPhone pro max 12

**Reply by isontheline** [reply]
To jump through a SSH server, a possible solution is to use DPF : https://webssh.net/documentation/help/networking/dynamic-port-forwarding/
